lib/pdk/util.rb

```diff
@@ -132,6 +132,14 @@ def module_root
     end
     module_function :module_root
 
+    # The module's fixtures directory for spec testing
+    # @return [String] - the path to the module's fixtures directory
+    def module_fixtures_dir
+      dir = module_root
+      File.join(module_root, 'spec', 'fixtures') unless dir.nil?
```

Comment:
Adding a PR note here (won't stop merging) `<root>/spec/fixtures` is hardcoded here and (probably) comes from the `puppetlabs_spec_helper` gem

https://github.com/puppetlabs/puppetlabs_spec_helper/blob/9d66ff58eba8bd4e21bb31c638a6ef6e9ac8cd14/lib/puppetlabs_spec_helper/tasks/fixtures.rb#L268-L363

IIRC There is no way to override this, so hardcoding here is fine.